### PR TITLE
lose weight by move single used function inline

### PR DESCRIPTION
**Pull request by bluebill1049**

### CODE REVIEW

File: src/logic/mapIds.test.ts [hunk 1]
```diff
@@ -1,22 +0,0 @@
-import { appendId } from './mapIds';
-
-jest.mock('../logic/generateId', () => ({
```
Comment (pmaier983) [inline src/logic/mapIds.test.ts:3]: I don't love removing tests if we are still using the functions. Would it be too much a size burden to export appendId?

Comment (bluebill1049) [reply src/logic/mapIds.test.ts:3]: Agree. Will try to cover the tests in usefieldarry

Comment (bluebill1049) [reply src/logic/mapIds.test.ts:3]: I confirm those tests are reduduant, as @keiya01 has already converted those in his tests. :) 
Screen Shot 2020-07-12 at 3 01 00 pm

**Comment by bluebill1049** [comment]
gain is too little, closing...

**Review by keiya01 — APPROVED** [review]
LGTM :)

**Review by JeromeDeLeon — APPROVED** [review]
LGTM! :cocktail: :cocktail:

**Comment by bluebill1049** [comment]
Thanks @JeromeDeLeon & @keiya01's review

**Pull request merged**